### PR TITLE
Estimate prioritization

### PR DESCRIPTION
**Pull request by atatmaja**

### CODE REVIEW

**Comment by atatmaja** [comment]
Note don't merge yet. Need to test more thoroughly.

**Comment by atatmaja** [comment]
Also @abeljoseph can you help me check this failing test, it's passing locally for me...

**Review by simonarocco — COMMENTED** [review]
looks good if has been tested. I'm guessing what needs to be more rigorously tested is the get prioritized estimates func itself. have we done this? I saw the hacky tests for it but should do manual testing too.

File: app/utils/get_filtered_records.py [hunk 1]
```diff
@@ -164,6 +168,15 @@ def date_filter(record, start_date=None, end_date=None):
 
     result = list(filter(lambda x: date_filter(x, start_date=start_date, end_date=end_date), result))
 
+    # TODO: Determine whether to update get_prioritized_estimates to work on dictionaries
+    # or keep everything in dataframes (don't want to have this conversion here long term)
+    if prioritize_estimates:
```
[the remark this answers is not in the document]
Comment (atatmaja) [reply app/utils/get_filtered_records.py:173]: Yeah, let's make this a ticket for now, since it would be better to have estimate prio in for re-arch

**Comment by atatmaja** [comment]
> looks good if has been tested. I'm guessing what needs to be more rigorously tested is the get prioritized estimates func itself. have we done this? I saw the hacky tests for it but should do manual testing too.

On my end, I need to more thoroughly test the endpoints (with some filter combinations), as I've edited the get_filtered_records utility function, which many endpts depend on. @rahularoradfs can comment on how he's tested the get prioritized estimates func itself

**Pull request merged**